main.js

```diff
@@ -3,18 +3,39 @@
 const fastify = require('fastify');
 
 const path = require('node:path');
+const fs = require('node:fs');
 
 const { Logger } = require('./src/logger.js');
 const http = require('./src/http.js');
 const ws = require('./src/ws.js');
-
 const { loadApplication } = require('./src/loader.js');
+
 const APPLICATION_PATH = path.join(process.cwd(), '../NodeJS-Application');
+const LOG_FOLDER_PATH = './log';
+
+const date = new Date().toISOString().substring(0, 10);
+const LOG_FILE = path.join(LOG_FOLDER_PATH, `${date}.log`);
+const logFileStream = fs.createWriteStream(LOG_FILE, { flags: 'a' });
+
+const streamForLogger = {
+  write(msg) {
+    process.stdout.write(msg);
+    const currentDate = new Date().toISOString().substring(0, 10);
+    if (currentDate !== this.date) {
```

Comment:
If you rely on using this -- I'd prefer to extract this to a Class.
In any case -- this needs to be extracted into /src/logger so as not to clutter the main entry point.

Reply:
@nechaido Done

Reply:
Not quite yet, see https://github.com/metatech-university/NodeJS-Fastify/pull/7/files#r1109789506.